make/modules/jdk.jpackage/Lib.gmk

```diff
@@ -44,27 +58,59 @@ endif
 
 JPACKAGE_OUTPUT_DIR := $(JDK_OUTPUTDIR)/modules/$(MODULE)/jdk/jpackage/internal/resources
 JPACKAGE_CXXFLAGS_windows := -EHsc -DUNICODE -D_UNICODE
+JPACKAGE_CFLAGS_windows := -DUNICODE -D_UNICODE
```

Comment:
Why is this change modifying Windows? I thought it was supposed to be a linux-only fix..?